covid19_sfbayarea/news/alameda.py

```diff
@@ -13,6 +13,29 @@
 logger = getLogger(__name__)
 
 
+# Maps the various forms of language names in the page's text to ISO language
+# codes so we can determine the language of a link.
+# TODO: Actually use these to handle more than just English links.
+LANGUAGES = {
+    'english': 'en',
+    'spanish': 'es',
+    'español': 'es',
+    'chinese': 'zh_HANS',
+    'chinese (simplified)': 'zh_HANS',   # 'zh_CN' is common, but less correct.
+    'chinese (traditional)': 'zh_HANT',  # 'zh_TW' is common, but less correct.
+    'korean': 'ko',
+    'vietnamese': 'vi',
+    'amharic': 'am',
+    'arabic': 'ar',
+    'farsi': 'fa',
+    'persian': 'fa',
+    'pashto': 'ps',
+    'urdu': 'ur',
+}
```

Comment:
This mapping isn’t strictly necessary at the moment, but we currently drop non-English links from the feed entirely. Having this mapping will allow someone to come through later a update the code to pick up non-English links and surface them in a meaningful way, whatever that might be.